bin/iam-token.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# the iam-token.sh scripts generate new IAM token
+# it relies on the following environment variables
+# IAM_CLIENT_ID client id value
+# IAM_CLIENT_SECRET client secret value
+# IAM_TOKEN output file name to store obtained IAM token
```

Comment:
Maybe we should rename it from `IAM_TOKEN` to `IAM_TOKEN_FILENAME`?

Reply:
Alan, please state it either way, I can't interpret maybe. To me `IAM_TOKEN` is totally fine, if you want it to be renamed please write this explicitly without question marks. SO far I leave it as is